src/web-auth/index.js

```diff
@@ -920,13 +946,25 @@ WebAuth.prototype.passwordlessVerify = function(options, cb) {
   );
 
   params = this.transactionManager.process(params);
+
   return this.client.passwordless.verify(params, function(err) {
     if (err) {
       return cb(err);
     }
-    return windowHelper.redirect(
-      _this.client.passwordless.buildVerifyUrl(params)
-    );
+
+    function doAuth() {
+      return windowHelper.redirect(
+        _this.client.passwordless.buildVerifyUrl(params)
+      );
+    }
+
+    if (typeof options.onRedirecting === 'function') {
+      return options.onRedirecting(function() {
+        return doAuth();
+      });
```

Comment:
Same here, this will always return `undefined` unless `return done()` is used when setting up the `onRedirecting` callback. I don't think this matters again, as the return value of `passwordlessVerify` isn't used.

Reply:
Yep, will remove.